app/web/src/components/ModelingDiagram/ModelingDiagram.vue

```diff
@@ -1322,10 +1323,7 @@ function onDragElementsMove() {
 
     // if we are going to move the element within a new parent we may need to adjust
     // the position to stay inside of it
-    if (newParent && newParent.def.componentId !== el.def.parentComponentId) {
-      // if dragging OUT of a parent onto the root, we want to make sure the element is not
-      // floating on top of another group
-
+    if (newParent) {
```

Comment:
Could we call this something like `parentOrCandidate` or something? the `newParent` naming tripped me up the first time I was reviewing this